src/common/config.cpp

```diff
@@ -116,6 +116,21 @@ void Config::initialize(ConfigParser const& cp) {
     config_["tsv-fields"] = tsvFields;
   }
 
+  // ensures factors backward compability whilst keeping the more user friendly CLI
```

Comment:
Heh, I had overlooked this misspelling when I was editing the code: `compability` -> `compatibility`.

Reply:
done